aui.views/src/AUI/Platform/ABaseWindow.h

```diff
@@ -362,11 +363,15 @@ class API_AUI_VIEWS ABaseWindow: public AViewContainer {
     virtual void hideTouchscreenKeyboardImpl();
 
 private:
+    void processTouchscreenKeyboardRequest();
+
     _weak<AView> mFocusedView;
     _weak<AView> mProfiledView;
     float mDpiRatio = 1.f;
-    bool mIgnoreTouchscreenKeyboardRequests = false; // to avoid flickering
 
+    ATouchscreenKeyboardState mKeyboardRequestedState = ATouchscreenKeyboardState::UNKNOWN;
+    bool mKeyboardRequestForce = false;
```

Comment:
Please add docs referencing `ATouchscreenKeyboardPolicy` or use `ATouchscreenKeyboardPolicy` instead of bool (better).